jme3-core/src/main/java/com/jme3/light/DirectionalLight.java

```diff
@@ -59,16 +59,39 @@ public class DirectionalLight extends Light {
      * Creates a DirectionalLight
      */
     public DirectionalLight() {
+        super();
+    }
+
+    /**
+     * Creates a DirectionalLight
+     * @param global if true, the light affects the entire tree from the root node,
+     * otherwise it only affects the children of the node in which it is attached.     
+     */
+    public DirectionalLight(boolean global) {
```

Comment:
i didn't make a setter for the global param, to make it defacto immutable.
This makes implementation much easier and performant, since we never need to check if the flag changed.